04_numeronym/madhavansu/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprint(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+func numeronyms(vals ...string) []string {
+	var strcopy []string
+	for _, val := range vals {
```

Comment:
After preallocating the size you can use the `position` that comes from `range` as 
```go
for pos, val := range vals
```
and assign the values as 
```go
strcopy[pos] = fmt.Sprint(first, length-2, last)
```

Reply:
As I said in the previous comment, this is another approach.